rules_building_block/execution_github_high_number_of_cloned_repos_from_pat.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2023/10/11"
+integration = ["github"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/10/11"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects a high number of repo clone events originating from a single personal access token within a 5 minute interval.
+"""
+from = "now-9m"
+index = ["logs-github.audit-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "High Number of Cloned GitHub Repos From PAT"
+risk_score = 21
+rule_id = "fb0afac5-bbd6-49b0-b4f8-44e5381e1587"
+severity = "low"
+tags = ["Domain: Cloud",
+        "Use Case: Threat Detection",
+        "Tactic: Execution",
+        "Rule Type: BBR",
+        "Data Source: Github"
+        ]
+timestamp_override = "event.ingested"
+type = "eql"
+building_block_type = "default"
```

Comment:
I learnt last week that all of these files are in alphabetic order. So `building_block_type` should go after `author` (for all the BBRs). Not sure about `bypass_bbr_timing`, and whether that one should go before `creation_date`.